File: testdata/lint/py/network.py
# network.py is a capability lint fixture (task 4.2): every line here is a
# genuine match for the network detection class table (spec 9). Comment
# based and dynamic import based limitations are exercised separately in
# falsenegatives.py, not mixed in here, so this file's detection count stays
# exactly one per table entry.

import requests
from requests import get
import httpx
from httpx import Client
import urllib
from urllib import request
import socket
from socket import socket as raw_socket
import aiohttp
from aiohttp import ClientSession

File: testdata/lint/js/network.ts
// network.ts is a capability lint fixture (task 4.1): every line here is a
// genuine match for the network detection class table (spec 9). Comment
// based and eval based limitations are exercised separately in
// falsenegatives.ts, not mixed in here, so this file's detection count stays
// exactly one per table entry.

import httpImport from "http";
import httpsImport from "https";
import netImport from "net";
import axios from "axios";
import { Agent } from "undici";

export async function fetchData() {
	const httpModule = require("http");
	const httpsModule = require("https");
	const netModule = require("net");
	return fetch("https://api.example.com");
}
